reference-data-proxy/src/v1/controllers/acbs.controller.ts

```diff
@@ -103,30 +131,63 @@ export const issueAcbsFacilityPOST = async (req: Request, res: Response) => {
   return res.status(400).send();
 };
 
-const createAcbsRecord = async (deal: any, bank: any) => {
-  if (deal) {
+/**
+ * Invoked Azure DOF using HTTP `POST` method.
+ * @param {Object} amendment Amendment object comprising facility ID and amends. A amendment at a time is processed.
+ * @returns {Object} DOF Response
+ */
+const amendAcbsFacility = async (amendment: object) => {
+  const hasAcceptablePayload = Object.prototype.hasOwnProperty.call(amendment, 'coverEndDate') || Object.prototype.hasOwnProperty.call(amendment, 'amount');
+
+  if (amendment && hasAcceptablePayload) {
     const response = await axios({
       method: 'post',
-      url: `${acbsFunctionUrl}/api/orchestrators/acbs`,
+      url: `${acbsFunctionUrl}/api/orchestrators/acbs-amend-facility`,
       data: {
-        deal,
-        bank,
+        amendment,
       },
-    }).catch((err: any) => err);
+    }).catch((e: any) => e);
+
     return response;
   }
   return {};
 };
 
-export const createAcbsRecordPOST = async (req: Request, res: Response) => {
+/**
+ * ACBS facility amendment entry function.
+ * Constructs acceptable payload by DOF.
+ * @param {Object} req Request
+ * @param {Object} res Response
+ * @return {Object} Response object with HTTP code as `status` and response as `data`.
+ */
+export const amendAcbsFacilityPost = async (req: Request, res: Response) => {
   if (req) {
```

Comment:
same as above - do we need this check here?

Reply:
Have rectified.